src/features/app/thanks/send/components/send.tsx

```diff
@@ -0,0 +1,298 @@
+    url: emojiHappyUrl,
+    label: "Happy",
+  },
+  {
+    value: "helpful",
+    url: emojiHelpfulUrl,
+    label: "Helpful",
+  },
+];
+const BUTTON_DATA = [
+  {
+    value: "1",
+    isSelected: false,
+    isDisabled: false,
+  },
+  {
+    value: "2",
+    isSelected: false,
+    isDisabled: false,
+  },
+  {
+    value: "5",
+    isSelected: false,
+    isDisabled: false,
+  },
+];
+
+type Emoji = {
+  value: string;
+  url: string;
+};
+
+type ThanksButton = {
+  value: string;
+  isSelected: boolean;
+  isDisabled: boolean;
+};
+
+export const Send = () => {
+  const [selectedEmoji, setSelectedEmoji] = useState<Emoji | null>(null);
+  const [thanksButtons, setThanksButtons] = useState<ThanksButton[]>(
+    BUTTON_DATA.map((button) => ({
+      ...button,
+      isDisabled: button.value > THANKS.toString(),
+    })),
+  );
+  const [sliderValue, setSliderValue] = useState(0);
+  const [isSent, setIsSent] = useState(false);
+
+  const handleSelectEmoji = (emoji: { value: string; url: string }) => {
+    setSelectedEmoji(emoji);
+  };
+
+  const handleSelectThanksButton = (value: string) => {
+    setThanksButtons((prev) =>
+      prev.map((button) =>
+        button.value === value
+          ? { ...button, isSelected: true }
+          : { ...button, isSelected: false },
+      ),
+    );
+  };
+
+  const handleSliderChange = (_event: Event, newValue: number | number[]) => {
+    setSliderValue(newValue as number);
+  };
+
+  const handleSendThanks = () => {
+    setIsSent(true);
+  };
+
+  // popover
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
+  const handleOpenPopover = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+  const handleClosePopover = () => {
+    setAnchorEl(null);
+  };
+  const open = Boolean(anchorEl);
+
+  return (
+    <Stack
+      spacing={"30px"}
+      sx={{
+        backgroundColor: "background.default",
+        padding: "30px 20px",
+        height: "calc(100svh - 44px)",
+      }}
+    >
+      <Stack alignItems="center" gap={"10px"}>
+        <Avatar
+          alt="staff1"
+          src={staff1Url}
+          sx={{
+            width: 100,
+            height: 100,
+            border: "2px solid",
+            borderColor: "secondary.main",
+          }}
+        />
+        <Typography variant="h2" fontSize={"26px"}>
+          John
+        </Typography>
+      </Stack>
+
+      <Card>
+        <CardContent sx={{ padding: "0" }}>
+          <Stack
+            direction="row"
+            flexWrap={"nowrap"}
+            spacing={"10px"}
+            overflow={"auto"}
+            padding={"10px"}
+            borderBottom={"1px solid #E0E0E0"}
+          >
+            {EMOJI_DATA.map((emoji) => (
+              <Stack key={emoji.value} width={"50px"} sx={{ flexShrink: 0 }}>
+                <button
+                  type="button"
+                  onClick={() =>
+                    handleSelectEmoji({
```

Comment:
他のコンポーネントと命名規則を統一したい

```suggestion
                    handleEmojiSelect({
```

Reply:
他のコンポーネント側を修正して命名規則は統一しました！